src/cpp/tests/automation/testthat/test-automation-sweave.R

```diff
@@ -1,8 +1,9 @@
 
 library(testthat)
 
-self <- remote <- .rs.automation.createRemote()
-client <- remote$client
+self <- remote <- .rs.automation.newRemote()
```

Comment:
It'd be nice to wrap this up into a helper function where we can control if tests are run all in a single RStudio session versus each having their own separate session.